src/modules/client/client.service.ts

```diff
@@ -0,0 +1,153 @@
+import { Skill } from './../skill/entities/skill.entity';
+import { Freelancer } from 'src/modules/freelancer/entities/freelancer.entity';
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Any, ArrayOverlap, In, Repository } from 'typeorm';
```

Comment:
"Any, ArrayOverlap, In" are not used as well

Reply:
+